src/core/layout/scrollable-list/scrollable-list-buttons.js

```diff
@@ -41,15 +39,9 @@ const createGelButton = (scene, item, title, state, prepTx) => {
         state: STATES.find(st => st === state),
     };
 
-    const callback = () => prepTx(item, title);
-
-    const clickEvent = eventBus.subscribe({
-        callback: handleClickIfVisible(gelButton, scene, callback),
-        channel: gelConfig.channel,
-        name: id,
-    });
-    scene.events.once("shutdown", clickEvent.unsubscribe);
-
+    const callback = () => gelButton.input.enabled && prepTx(item, title);
```

Comment:
The simplification of the code here is a good win, but maybe we need a different check to `gelButton.input.enabled`? perhaps check `panel.isInTouching()`?

Reply:
The problem with this is that then I have to have two different paths of logic to deal with an accessibility click and an actual click - which I'd rather avoid. I think I've got a way to do this, just going to try it